packaging/deb/postinst

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+set -e
+
+if ! grep -q '## OpenDistro Performance Analyzer' /etc/elasticsearch/jvm.options; then
+   CLK_TCK=`/usr/bin/getconf CLK_TCK`
+   echo >> /etc/elasticsearch/jvm.options
+   echo '## OpenDistro Performance Analyzer' >> /etc/elasticsearch/jvm.options
+   echo "-Dclk.tck=$CLK_TCK" >> /etc/elasticsearch/jvm.options
+   echo "-Djdk.attach.allowAttachSelf=true" >> /etc/elasticsearch/jvm.options
+   echo "-Djava.security.policy=file:///usr/share/elasticsearch/plugins/opendistro_performance_analyzer/pa_config/es_security.policy" >> /etc/elasticsearch/jvm.options
+fi
+
+IS_UPGRADE=false
+# Below codeblock is using the fact that postinst script is called with the most-recently configured version.
+# In other words, a fresh installed will be called like "postinst configure" with no previous version ($2 is null)
+if [ -z "$2" ]; then
+  # If $2 is null, this is an install
+  IS_UPGRADE=false
+else
+  # otherwise this is an upgrade
+  IS_UPGRADE=true
+fi
+
+#DEBHELPER#
+
+if [ "x$IS_UPGRADE" != "xtrue" ]; then
+    if command -v systemctl > /dev/null; then
+        echo '# Enabling opendistro performance analyzer to start and stop along with elasticsearch.service'
+        systemctl daemon-reload
+        systemctl enable opendistro-performance-analyzer.service || true
```

Comment:
Should we enable the service on upgrade if it was disabled in the previous version?

Reply:
Correct me if I'm wrong, but seems to me this block (line 26 to line 36) is only handling the case of a fresh install. If we are handling an upgrade, my opinion is that we should leave the previous settings as they are, which is what we are doing in this script.